services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -397,6 +398,17 @@ public Response updateConsent(final @PathParam("clientId") String clientId,
                                   final ConsentRepresentation consent) {
         return upsert(clientId, consent);
     }
+    
+    @Path("/totp/remove")
```

Comment:
Bruno, when you take over this code, change the @Path to "/totp".  Realized that the "/remove" part of the path is redundant since this is a DELETE endpoint.